odxtools/cli/main.py

```diff
@@ -2,13 +2,21 @@
 # Copyright (c) 2021-2022 MBition GmbH
 
 import argparse
-
-from . import list
-from . import browse
-from . import snoop
-from . import find
+import importlib
 
 from ..version import __version__ as odxtools_version
+from .dummy_sub_parser import DummyTool
+
+# import the tool modules which can be loaded. if a tool
+# can't be loaded, add a dummy one
+tool_modules = []
+for tool_name in ["list", "browse", "snoop", "find"]:
+    try:
+        tool_modules.append(
+            importlib.import_module(f".{tool_name}", package="odxtools.cli"))
+    except Exception as e:
```

Comment:
```suggestion
    except ImportError as e:
```

Reply:
I'm not so sure about this: if the exception thrown is not an ImportError, it should be ignored as well, right? that said, I'm not sure if this can happen with `importlib.import_module`. any ideas?